### PR TITLE
blog: introducing myself to the community

### PR DESCRIPTION
**Pull request by rabajaj0509**

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
@tbrisker can you review and merge? looks good to me, and I'd like this out asap to match my post :)

File: _posts/2019-02-25-introducing-myself.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+tags:
+- foreman
+- community
+- whoami
+---
+
+I would like to take this opportunity to introduce myself, I am Rahul Bajaj and I 
+have been working as a Software Engineer at Red Hat for a year and a half now. I 
+am the co-maintainer of Foreman Discovery Plugin and also contribute to repos like 
+theforeman/foreman, hammer-cli-foreman, forklift, and pulp. Currently I am working
```
Comment (tbrisker) [inline _posts/2019-02-25-introducing-myself.md]:
```suggestion
Foreman core, hammer, forklift, and pulp. Currently I am working 
```

**Review by tbrisker — COMMENTED** [review]
just a couple of inline suggestions :)

File: _posts/2019-02-25-introducing-myself.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+I have always been interested in community where I keep meeting new like-minded 
+people that share the same interests as mine. Talking of the Foreman community, 
+I run the Foreman Pune Meetups chapter, where my main aim is to introduce new 
+contributors to the project and ask them about their experience working with it. 
+
+I like to speak about my work and have done so in various conferences like 
+[TheDevTheory Conf](https://thedevtheory.com/#rahulbajaj), [DevConf.in](https://devconfin2018.sched.com/speaker/rahul_bajaj.1y7e64ly), 
+[Freenode #live](https://freenode.live/), etc. As excinting it is to speak at a conference, 
+I take emense pleasure in volunteering too and have been a part of conferences
```
Comment (tbrisker) [inline _posts/2019-02-25-introducing-myself.md]:
```suggestion
I take immense pleasure in volunteering too and have been a part of conferences 
```

**Comment by rabajaj0509** [comment]
Thanks for reviewing @tbrisker, I have made the changes as asked :)

**Review by tbrisker — APPROVED** [review]
Thanks @rahulbajaj0509 and good luck!

**Pull request merged**